File: AI/RAI/java-cgdk/src/main/java/MyStrategy.java
import model.*;

import static java.lang.StrictMath.PI;

public final class MyStrategy implements Strategy {
    @Override
    public void move(Tank self, World world, Move move) {
        move.setLeftTrackPower(-1.0D);
        move.setRightTrackPower(1.0D);
        move.setTurretTurn(PI);
        move.setFireType(FireType.PREMIUM_PREFERRED);
    }

    @Override
    public TankType selectTank(int tankIndex, int teamSize) {
        return TankType.MEDIUM;
    }
}

File: trunk/AI/RAI/python3-cgdk/MyStrategy.py
from math import *
from model.FireType import FireType
from model.TankType import TankType

class MyStrategy:
    def move(self, me, world, move):
        move.left_track_power = -1.0
        move.right_track_power = 1.0
        move.turret_turn = pi
        move.fire_type = FireType.PREMIUM_PREFERRED

    def select_tank(self, tank_index, team_size):
        return TankType.MEDIUM